### PR TITLE
Update black

### PR DESCRIPTION
**Pull request by antonpirker**
.. and reformat code with new black.

### CODE REVIEW

File: tests/utils/test_general.py [hunk 1]
```diff
@@ -31,19 +31,23 @@
     def test_safe_repr_never_broken_for_strings(x):
         r = safe_repr(x)
         assert isinstance(r, text_type)
-        assert u"broken repr" not in r
+        assert "broken repr" not in r
 
 
 def test_safe_repr_regressions():
+    # fmt: off
```
Comment (sl0thentr0py) [inline tests/utils/test_general.py:38]: why are these needed?

Comment (antonpirker) [reply tests/utils/test_general.py:38]: So that black does not remove the u"" from the strings and breaks our tests.

Comment (antonpirker) [reply tests/utils/test_general.py:38]: You can "turn off" black formatting for parts of the code.

Comment (antonpirker) [reply tests/utils/test_general.py:38]: strings containing only ascii characters do not need u"" in python 2.7. Only strings containing unicode chars need the u"". (hence "u") 

I think Python (2) did a really bad job in adding unicode support to the language
